modules/dqt/jsx/components/expansionpanels.js

```diff
@@ -0,0 +1,93 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+
+const Panel = (props) => {
+  const [active, setActive] = useState(props.alwaysOpen);
+
+  const styles = {
+    accordion: {
+      default: {
+        width: '100%',
+        padding: '18px',
+        outline: 'none',
+        color: '#246EB6',
+        fontSize: '15px',
+        cursor: props.alwaysOpen ? 'default' : 'pointer',
+        textAlign: 'center',
+        backgroundColor: '#fff',
+        border: '1px solid #246EB6',
+        transition: '0.4s',
+      },
+      active: {
+        color: '#fff',
+        textAlign: 'left',
+        backgroundColor: '#246EB6',
+      },
+    },
+    panel: {
+      default: {
+        display: 'none',
+        padding: '20px 18px',
+        backgroundColor: '#fff',
+        border: '1px solid #246EB6',
+        overflow: 'hidden',
+      },
+      active: {
+        display: 'block',
+        margin: '0 0 10px 0',
+      },
+    },
+  };
+
+  const handleExpansionClick = () => {
+    if (props.alwaysOpen) return;
+    setActive((active) => !active);
+  };
+
+  const styleAccordion = {
+    ...styles.accordion.default,
+    ...(active ? styles.accordion.active : {}),
+  };
+
+  const stylePanel = {
+    ...styles.panel.default,
+    ...(active ? styles.panel.active : {}),
+  };
+
+  return (
+    <>
```

Comment:
?

Reply:
@driusan using `<></>` is the newer way in react instead of `<React.Fragment></React.Fragment>` and to prevent an unnecessary div being needed for multiple elements that are being returned.